packages/validator/package.json

```diff
@@ -24,24 +24,28 @@
   },
   "dependencies": {
     "@ibm-cloud/openapi-ruleset": "1.23.1",
+    "@ibm-cloud/openapi-ruleset-utilities": "1.4.0",
     "@stoplight/spectral-cli": "^6.13.1",
     "@stoplight/spectral-core": "^1.19.1",
     "@stoplight/spectral-parsers": "^1.0.4",
+    "@stoplight/spectral-ref-resolver": "^1.0.4",
     "ajv": "^8.17.1",
     "chalk": "^4.1.2",
     "commander": "^10.0.1",
+    "console-table-printer": "^2.12.1",
     "find-up": "5.0.0",
     "globby": "^11.0.4",
     "js-yaml": "^3.14.1",
     "json-dup-key-validator": "^1.0.3",
     "lodash": "^4.17.21",
+    "nimma": "^0.7.0",
```

Comment:
`"@stoplight/spectral-ref-resolver"` and `"nimma"` were already installed dependencies through Spectral, but this makes it obvious that we need them.